doc/auth.md

```diff
@@ -0,0 +1,71 @@
+# Enable Authentication
+
+Client can communicate with Pravega in a more secure way by enabling Authentication.
```

Comment:
This paragraph should be revised. It is ambiguous and vague. It can be revised to something along the lines of the following: 

"Pravega supports pluggable authentication and authorization (referred to as `auth` for short). By default, the `PasswordAuthHandler` plugin is installed on the system. New plugins can be deployed in the system at installation time, by supplying additional plugins as explained [here](http://...). To use the default `PasswordAuthHandler` plugin for `auth`, you must supply the plugin a file containing user credentials and thir Access Control List (ACL). To create the file...."

Reply:
Done.